tests/encodeinput.cpp

```diff
@@ -176,29 +177,54 @@ EncodeOutput* EncodeOutput::create(const char* outputFileName, int width , int h
     if(ext==NULL)
         return NULL;
     ext++;//h264;264;jsv;avc;26l;jvt;ivf
-    if(strcasecmp(ext,"h264")==0 ||
-        strcasecmp(ext,"264")==0 ||
-        strcasecmp(ext,"jsv")==0 ||
-        strcasecmp(ext,"avc")==0 ||
-        strcasecmp(ext,"26l")==0 ||
-        strcasecmp(ext,"jvt")==0 ) {
+
+    if (!strcasecmp(codecName, "AVC")) {
```

Comment:
This is kind of weird, if we're guessing based on the extension, then we should not give the option for codecName.  The problem then will be where two codecs can produce the same extension file, i.e. vpx both create ivf files.